types.ts

```diff
@@ -28,11 +28,11 @@ export interface FeatureData {
     compat_features?: string[];
 }
 
-type browserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
+export type BrowserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
```

Comment:
If you're going to export this stuff, then it needs a description in the schema. You'll need to do something like this, for each new exported type:

```suggestion
/** Browser identifiers (e.g., "chrome", "firefox", "safari") */
export type BrowserIdentifier = "chrome" | "chrome_android" | "edge" | "firefox" | "firefox_android" | "safari" | "safari_ios";
```